tests/main_test.ts

```diff
@@ -191,6 +191,15 @@ Deno.test("/foo/:path*", async () => {
   assert(body.includes("bar/baz"));
 });
 
+Deno.test("decodes parameters", async () => {
+  const resp = await router(new Request("https://fresh.deno.dev/this has spaces"));
+  assert(resp);
+  assertEquals(resp.status, Status.OK);
+  const body = await resp.text();
+  console.log(body)
+  assert(body.includes("this has spaces"));
```

Comment:
I think an assertion in here is missing to check whether it decodes the params. However having said that I don't think this actually calls the `decodeParams` function defined in the server context. Might want to change the location of this test

Reply:
> (copy of https://github.com/denoland/fresh/pull/625#discussion_r965659685)

This issue isn't about query parameters, but path parameters, which the test does assert correctly.